nixos/modules/programs/environment.nix

```diff
@@ -21,6 +21,7 @@ in
         PAGER = mkDefault "less -R";
         EDITOR = mkDefault "nano";
         XCURSOR_PATH = [ "$HOME/.icons" ];
+        XDG_CONFIG_DIRS = [ "/etc/xdg" ]; # needs to be before profile-relative paths to allow changes through environment.etc
```

Comment:
Perhaps explain the why as in the directories are ordered by priority.

Reply:
Does not the need to be first already imply the precedence relation? If you come up with a clearer comment, feel free to change it.